scripts/lib/dependencies.ts

```diff
@@ -67,9 +67,71 @@ export interface Dependency {
   getLatestVersion(): Promise<string | AlpineLimaISOVersion>
 }
 
+/**
+ * Types that implement UnreleasedChangeMonitor can tell you whether
+ * there have been any changes in their repository since their last release.
+ */
+export interface UnreleasedChangeMonitor {
+  hasUnreleasedChanges(): Promise<HasUnreleasedChangesResult>
+}
+
+export type HasUnreleasedChangesResult = {latestReleaseTag: string, hasUnreleasedChanges: boolean};
+
+type GithubRelease = Awaited<ReturnType<Octokit['rest']['repos']['listReleases']>>['data'][0];
+
+async function getLatestPublishedRelease(githubOwner: string, githubRepo: string): Promise<GithubRelease> {
+  const response = await getOctokit().rest.repos.listReleases({ owner: githubOwner, repo: githubRepo });
+
+  for (const release of response.data) {
+    if (release.published_at !== null) {
+      return release;
+    }
+  }
```

Comment:
So, roughly, `const release = response.data.find(release => release.published_at)`
(Your code is fine, I'm just checking my understanding.)

Reply:
Yes, I think so. I'll change it to this; it's nice to save some lines.

Reply:
Hmm actually I think I'll leave it. Using `find` is about the same because of the need to handle the case where the `published_at` key is `null` for each release.